data/scene/missions/juno/juno/juno.mod

```diff
@@ -126,16 +126,19 @@ return {
         Name = "JunoTrail",
         Parent = "JupiterBarycenter",
         Renderable = {
-            Type = "RenderableTrailTrajectory",
-            Translation = {
-                Type = "SpiceTranslation",
-                Body = "JUNO",
-                Observer = "JUPITER BARYCENTER"
-            },
-            Color = { 0.70, 0.50, 0.20 },
-            StartTime = "2016 JUN 28",
-            EndTime = "2016 APR 01",
-            SampleInterval = 3600
+            Type = "RenderableTrail",
```

Comment:
This is removing the new specification and replacing it with the old ...   it shouldn't be changed at all